climakitae/core/data_interface.py

```diff
@@ -559,12 +569,23 @@ class DataParameters(param.Parameterized):
         shorthand alias to DataInterface.geographies
     _geography_choose: dict
         shorthand alias to Boundaries.boundary_dict()
+    _warming_level_times: pd.DataFrame
+        shorthand alias to DataInterface.warming_level_times
     colormap: str
         default colormap to render the currently selected data
     scenario_options: list of strs
         list of available scenarios (historical and ssp) for selection
     variable_options_df: pd.DataFrame
         filtered variable descriptions for the downscaling_method and timescale
+    warming_level: array
+        global warming level(s)
+    warming_level_window: integer
+        Years around Global Warming Level (+/-) \n (e.g. 15 means a 30yr window)
+    approach: str, "Warming Level" or "Time"
+        how do you want the data to be retrieved?
+    warming_level_months: array
+        which months to use for computing warming levels?
```

Comment:
How come approach and here are questions?

Reply:
fixed